diff --git a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicBlurPostProcessor.java b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicBlurPostProcessor.java
index dcc4948..718f205 100644
--- a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicBlurPostProcessor.java
+++ b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicBlurPostProcessor.java
@@ -2,8 +2,8 @@ package iyegoroff.imagefilterkit.nativeplatform.scriptintrinsic;
 
 import android.content.Context;
 import android.graphics.Bitmap;
-import androidx.renderscript.Element;
-import androidx.renderscript.ScriptIntrinsicBlur;
+import android.renderscript.Element;
+import android.renderscript.ScriptIntrinsicBlur;
 
 import com.facebook.cache.common.CacheKey;
 import com.facebook.cache.common.SimpleCacheKey;
diff --git a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve3x3PostProcessor.java b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve3x3PostProcessor.java
index 64b4ba0..8023e4e 100644
--- a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve3x3PostProcessor.java
+++ b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve3x3PostProcessor.java
@@ -2,8 +2,8 @@ package iyegoroff.imagefilterkit.nativeplatform.scriptintrinsic;
 
 import android.content.Context;
 import android.graphics.Bitmap;
-import androidx.renderscript.Element;
-import androidx.renderscript.ScriptIntrinsicConvolve3x3;
+import android.renderscript.Element;
+import android.renderscript.ScriptIntrinsicConvolve3x3;
 
 import com.facebook.cache.common.CacheKey;
 import com.facebook.cache.common.SimpleCacheKey;
diff --git a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve5x5PostProcessor.java b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve5x5PostProcessor.java
index 78e2863..ae14195 100644
--- a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve5x5PostProcessor.java
+++ b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/nativeplatform/scriptintrinsic/ScriptIntrinsicConvolve5x5PostProcessor.java
@@ -2,8 +2,8 @@ package iyegoroff.imagefilterkit.nativeplatform.scriptintrinsic;
 
 import android.content.Context;
 import android.graphics.Bitmap;
-import androidx.renderscript.Element;
-import androidx.renderscript.ScriptIntrinsicConvolve5x5;
+import android.renderscript.Element;
+import android.renderscript.ScriptIntrinsicConvolve5x5;
 
 import com.facebook.cache.common.CacheKey;
 import com.facebook.cache.common.SimpleCacheKey;
diff --git a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptCompositionPostProcessor.java b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptCompositionPostProcessor.java
index 47348de..7742965 100644
--- a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptCompositionPostProcessor.java
+++ b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptCompositionPostProcessor.java
@@ -4,8 +4,8 @@ import android.content.Context;
 import android.graphics.Bitmap;
 import android.graphics.Canvas;
 import android.graphics.Paint;
-import androidx.renderscript.Allocation;
-import androidx.renderscript.RenderScript;
+import android.renderscript.Allocation;
+import android.renderscript.RenderScript;
 
 import com.facebook.cache.common.CacheKey;
 import com.facebook.common.references.CloseableReference;
diff --git a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptSingularPostProcessor.java b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptSingularPostProcessor.java
index 511196d..9d978e5 100644
--- a/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptSingularPostProcessor.java
+++ b/node_modules/react-native-image-filter-kit/android/src/main/java/iyegoroff/imagefilterkit/utility/RenderscriptSingularPostProcessor.java
@@ -2,8 +2,8 @@ package iyegoroff.imagefilterkit.utility;
 
 import android.content.Context;
 import android.graphics.Bitmap;
-import androidx.renderscript.Allocation;
-import androidx.renderscript.RenderScript;
+import android.renderscript.Allocation;
+import android.renderscript.RenderScript;
 
 import org.json.JSONObject;
 
diff --git a/node_modules/react-native-image-filter-kit/examples/CustomNativeFilter/local_modules/react-native-image-filter-kit-haze-removal/android/src/main/java/iyegoroff/imagefilterkit/hazeremoval/HazeRemovalPostProcessor.java b/node_modules/react-native-image-filter-kit/examples/CustomNativeFilter/local_modules/react-native-image-filter-kit-haze-removal/android/src/main/java/iyegoroff/imagefilterkit/hazeremoval/HazeRemovalPostProcessor.java
index 3ee09ea..74b2a9a 100644
--- a/node_modules/react-native-image-filter-kit/examples/CustomNativeFilter/local_modules/react-native-image-filter-kit-haze-removal/android/src/main/java/iyegoroff/imagefilterkit/hazeremoval/HazeRemovalPostProcessor.java
+++ b/node_modules/react-native-image-filter-kit/examples/CustomNativeFilter/local_modules/react-native-image-filter-kit-haze-removal/android/src/main/java/iyegoroff/imagefilterkit/hazeremoval/HazeRemovalPostProcessor.java
@@ -3,7 +3,7 @@ package iyegoroff.imagefilterkit.hazeremoval;
 import android.content.Context;
 import android.graphics.Bitmap;
 import android.graphics.Color;
-import androidx.renderscript.Float4;
+import android.renderscript.Float4;
 
 import com.facebook.cache.common.CacheKey;
 import com.facebook.cache.common.SimpleCacheKey;
